modular_newhorizons/master_files/code/modules/background/origins/origins/human/sol.dm

```diff
@@ -0,0 +1,207 @@
+	name = "Jupiter"
+	desc = "Jupiter as a gas giant lacks a proper habitable 'surface', instead, all of its installations are in orbit around it or on the upper atmosphere of its expanse. While the majority of the orbital facilities are refueling stations or research outposts, all either government or corporate gas refineries, the most prominent is the 'Helios' Habitation Dock, once a simple Protectorate dockyard, now its an Idris owned tourist hotspot, supported by the fact that the habitation dock's residential levels turned into a luxury hotel, Helios hosts many scheduled events along its weekly terran cycles, looping the itinerary every week, which draws in many tourists and celebrities. For the average tourist, they will likely be captivated by the natural wonders of Jupiter such as the Red Spot or the Hexagon, Helios even has telescope theaters, to allow high-resolution public viewings of Jupiters great storms in many different visuals. And for the less visually impressed tourist or even for the permanent hotel residents, Helios is still a top of the line Idris resort, offering many luxury and entertainment options.\
+	The three major inhabited moons of Jupiter - Callisto, Ganymede, and Europa - are key Solar worlds. Callisto produces much of the food the Solar Colonies consume, Ganymed was once considered as a Protectorate port, but its rights were eventually sold to Nanotrasen, and Europa is a major research hub. The Jovian Moons are home to a remarkable diversity of humanity, and life from almost anywhere in known space can be found in their ports."
+	possible_accents = list(ACCENT_JUPITER, ACCENT_EUROPA, ACCENT_CALLISTO)
+	possible_citizenships = CITIZENSHIPS_SOLARIAN
+	possible_religions = RELIGIONS_SOLARIAN
+
+/singleton/origin_item/origin/pluto
+	name = "Pluto"
+	desc = "One of the last bodies in the Sol System to be colonized, Pluto was originally not targetted for any colonization effort, until colonization rights were purchased at a low price from the Protectorate by the upstart 'Evergreen Express' courier company to use as a staging ground for its services. This small company´s plans changed when large deposits of Helium-3 were discovered on Pluto and its moons, boosting its asset value by a large margin.\
+	ow, Pluto's economy is centered around Helium-3 mining and refinement and Evergreen Express has long rebranded to 'Charon Shipping Services', with its largest warehouse and dock being on the moon of Charon. A majority of Pluto´s legacy residents are offworlders as a result of the constant Zero-G work that they had to endure for most of their lives.\
+	While Pluto is a fuel world by resources, to the point where Charon Shipping has signed delivery agreements with several corporations and even some Protectorate posts, its also considered home to some of the best pilots in the System due to The Company´s own education systems orienting 'future employees' to know the basics of EVA and Piloting."
```

Comment:
Just some fixes to the description here.
>One of the last bodies in the Sol System to be colonised, Pluto was originally not targeted for any colonisation effort, until colonisation rights were purchased at a low price from the Protectorate by the upstart 'Evergreen Express' courier company to use as a staging ground for its services. This small company's plans changed when large deposits of Helium-3 were discovered on Pluto and its moons, boosting its asset value by a large margin.\
	Now, Pluto's economy is centred around Helium-3 mining and refinement and Evergreen Express has long rebranded to 'Charon Shipping Services', with its largest warehouse and dock being on the moon of Charon. A majority of Pluto's legacy residents are offworlders as a result of the constant Zero-G work that they had to endure for most of their lives.\
	While Pluto is a fuel world by resources, to the point where Charon Shipping has signed delivery agreements with several corporations and even some Protectorate posts, its also considered home to some of the best pilots in the system due to The Company's own education systems orienting 'future employees' to know the basics of EVA and Piloting.

Reply:
And done